examples/list_of_lists/test/test.ts

```diff
@@ -0,0 +1,535 @@
+import { Principal } from '@dfinity/principal';
+import { run_tests, Test } from 'azle/test';
+// import { get_tests } from 'azle/examples/list_of_lists/test/tests';
+import { execSync } from 'child_process';
+import { createActor } from './dfx_generated/list_of_lists';
+import { deepEqual } from 'fast-equals';
+import { Opt } from 'azle';
+
+const list_of_lists_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
+    agentOptions: {
+        host: 'http://127.0.0.1:8000'
+    }
+});
+
+function arrEqual<T>(a: DeepArray<T>, b: DeepArray<T>): boolean {
```

Comment:
Can you put this stuff below the tests? And this should move over into the `tests.ts` file.

Reply:
I have it that way in azle